backend/LexBoxApi/GraphQL/OrgMutations.cs

```diff
@@ -52,6 +53,69 @@ public class OrgMutations
         return org;
     }
 
+    [Error<DbError>]
+    [Error<NotFoundException>]
+    [UseMutationConvention]
+    [UseFirstOrDefault]
+    [UseProjection]
+    public async Task<IQueryable<Organization>> AddProjectToOrg(
+        LexBoxDbContext dbContext,
+        IPermissionService permissionService,
+        Guid orgId,
+        Guid projectId)
+    {
+        var org = await dbContext.Orgs.FindAsync(orgId);
+        NotFoundException.ThrowIfNull(org);
+        permissionService.AssertCanEditOrg(org);
```

Comment:
I think we've now agreed ([last two comments](https://github.com/sillsdev/languageforge-lexbox/issues/788#issuecomment-2162538984) on the issue) that you don't need to be an org admin to add or remove a project; you just need to be an org member.

Reply:
Yep, will do that on Monday. Also, thanks for catching that I hadn't actually pushed the commit with the UI change in it.

Reply:
UI change in commit e74cde75, and permissions change in commit 6c84f203.